dashboard/app/models/pd/async_workshop_handler.rb

```diff
@@ -0,0 +1,52 @@
+# An SQS messages handler for asynchronous workshop jobs,
+# notably wrapping up a workshop when it ends (sending emails, generating reports, etc.)
+class Pd::AsyncWorkshopHandler
+  ACTIONS = [
+    ACTION_CLOSE = 'close'
```

Comment:
Should this be close or end? I see two different verbs, are they doing the same thing?